routing/routes.go

```diff
@@ -218,6 +218,26 @@ func NewRoutes() (r *Routes) {
 			handlers.IstioConfigList,
 			true,
 		},
+		// swagger:route GET /istio config istioConfigListAll
+		// ---
+		// Endpoint to get the list of Istio Config of all namespaces
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http, https
+		//
+		// responses:
+		//      500: internalError
+		//      200: istioConfigList
+		//
+		{
+			"IstioConfigListAll",
+			"GET",
+			"/api/istio",
```

Comment:
Perhaps it could be better to narrow the endpoint to "/api/istio/config" as there are other endpoints like "/api/istio/permissions" or "/api/istio/certs".

It's just a naming suggestion to follow a similar convention here.